src/analyticsdf/analyticsdataframe.py

```diff
@@ -335,3 +335,32 @@ def update_response_poly_categorical(self, predictor_name: str = None, betas: di
                 self.response_vector = numeric_vals[predictor_name]
             else:
                 self.response_vector += numeric_vals[predictor_name]
+
+    def update_predictor_exp(self, rate: float = None, 
+                             size: tuple = None,
```

Comment:
Is tuple the ideal choice?

Reply:
Since the rest of the functions all use `list` or `np.array` as the input.

Reply:
Thanks for the feedback! 
I was thinking about giving the option of defining output shape, e.g. multi-dimensional output. But I just realized that'd require a significant more complex function. We should indeed go with the simple version for now.
Also I think the size should be the number of observations by default. I will just get rid of this argument `size` for now.